string model1
string link1
string model2
string link2
bool connect # true to connect, false to disconnect